reducer/src/main/java/com/graphicsfuzz/reducer/ReductionDriver.java

```diff
@@ -132,8 +158,23 @@ public ReductionDriver(ReducerContext context,
       corePasses.add(pass);
       cleanupPasses.add(pass);
     }
-    this.passManager = new SystematicReductionPassManager(initialPasses, corePasses, cleanupPasses);
+    return new SystematicReductionPassManager(initialPasses, corePasses, cleanupPasses);
+  }
+
+  private static IReductionPassManager getLiteralsToUniformsPassManager(
+      ReducerContext context,
+      boolean verbose) {
+    final List<IReductionPass> initialPasses = new ArrayList<>();
+    final List<IReductionPass> corePasses = new ArrayList<>();
+    final List<IReductionPass> cleanupPasses = new ArrayList<>();
```

Comment:
I would either initialize this to Arrays.asList(new SystematicReductionPass(...)), or just pass Arrays.asList(...) where you currently use cleanupPasses.